packages/common/src/services/audius-backend/AudiusBackend.ts

```diff
@@ -2093,7 +2394,7 @@ export const audiusBackend = ({
         : ''
       // TODO: withRemix, withTrending, withRewards are always true and should be removed in a future release
       const notifications = await fetch(
-        `${identityServiceUrl}/notifications?${limitQuery}${timeOffsetQuery}${handleQuery}${withDethronedQuery}&withTips=true&withRewards=true&withRemix=true&withTrendingTrack=true`,
+        `${identityServiceUrl}/notifications?${limitQuery}${timeOffsetQuery}${handleQuery}${withDethronedQuery}}&withTips=true&withRewards=true&withRemix=true&withTrendingTrack=true`,
```

Comment:
is there a libs method for this btw, or we just doing pure manual for this? also do we need withRemix, withTrending, withRewards needed these days?

Reply:
just manually - the whole idea before was that notifications was audius client specific and not not external consumption - so we never added to libs